content/covidcast/data-anomalies.md

```diff
@@ -0,0 +1,11 @@
+---
+title: COVIDcast Export Data
```

Comment:
This is still the export data page info

Reply:
my mistake, copy pasted a file and edited the original one